src/Notifier.php

```diff
@@ -105,9 +111,13 @@ public function buildNotice($exc)
             ],
             'os' => php_uname(),
             'language' => 'php '.phpversion(),
+            'rootDirectory' => $this->opt['rootDirectory'] ?: $_SERVER['DOCUMENT_ROOT'] ?: null,
```

Comment:
`$_SERVER['DOCUMENT_ROOT']` is bad default value for the `rootDirectory` option.

1. In CLI it has no meaning.
2. Airbrake users hardly ever store their code in DOCUMENT_ROOT